### PR TITLE
Padding setting of link button & nesting structure of _AccessibleButton.scss

### PR DESCRIPTION
**Pull request by luixxiul**
Signed-off-by: Suguru Hirahara <luixxiul@users.noreply.github.com>

type: task

This PR sets `padding: 0` by default to the link button and removes settings which become redundant thanks to it. Since there are fairly complex dependencies, this PR removes only ones which can be removed safely.

This PR also applies clear nesting structure on `_AccessibleButton.scss`, so that racing conditions should not appear.

---
This change is marked as an *internal change* (Task), so will not be included in the changelog.

Preview: https://pr8091--matrix-react-sdk.netlify.app
⚠️ Do you trust the author of this PR? Maybe this build will steal your keys or give you malware. Exercise caution. Use test accounts.

### CODE REVIEW

File: res/css/views/elements/_AccessibleButton.scss
Comment (luixxiul) [inline res/css/views/elements/_AccessibleButton.scss]: Note this has not been `&.mx_AccessibleButton_disabled`.

**Review by t3chguy — APPROVED** [review]
Looks sane, will hold off until after RC branches are cut to merge though

**Comment by luixxiul** [comment]
Thanks for checking, I appreciate it (as there isn't an automatic test to ensure no regression)

**Comment by t3chguy** [comment]
You need to pull in latest develop to get the tests passing

**Review by t3chguy — APPROVED** [review]
Thanks!!

**Pull request merged**
